spec/index.bs

```diff
@@ -1850,7 +1865,29 @@ interface GPUQueue {
 GPUQueue includes GPUObjectBase;
 </script>
 
- - {{GPUQueue/copyImageBitmapToTexture()}}:
+{{GPUQueue/writeBuffer()}} takes the `sourceData` contents starting from the offset `sourceOffset` and
+schedules a write operation of these contents to the `destination` buffer in the queue.
+The operation does nothing and produces an error if any of the following is true:
+
+ - `destination` texture wasn't created with `GPUTextureUsage.COPY_DST` flag in {{GPUTextureDescriptor/usage}}.
```

Comment:
`destination` can also be a GPUTexture? :)

Reply:
I messed up some copy pasting :sweat_smile: